app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -97,7 +97,7 @@
   </head>
   <%= tag.body class: body_css_classes do %>
     <%= javascript_tag nonce: true do -%>
-      document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');
+      document.body.className += ('noModule' in HTMLScriptElement.prototype ? ' js-enabled govuk-frontend-supported' : '');
```

Comment:
I think we should avoid moving the `js-enabled` class here to help keep the page template consistent with the design system - https://design-system.service.gov.uk/styles/page-template/, this would also help show that JS is enabled in the template, but modules are not supported (Grade X browsers)

Although it will likely be more work, we would likely want to update the CSS instead to use the `govuk-frontend-supported` class.

This was the approach used for the upgrade of govuk-frontend from v4 to v5:
- [Tabs govuk-frontend v4 using js-enabled](https://github.com/alphagov/govuk-frontend/blob/support/4.x/package/govuk/components/tabs/_index.scss#L44)
- [Tabs govuk-frontend v5 using govuk-frontend-supported](https://github.com/alphagov/govuk-frontend/blob/953f15903426a8dc508592ed7dad90a253b058d4/packages/govuk-frontend/src/govuk/components/tabs/_index.scss#L48)

Places where this may need to be updated - https://github.com/search?q=repo%3Aalphagov%2Fgovuk_publishing_components%20js-enabled&type=code

Reply:
Thanks @MartinJJones - I will look into this - `.js-enabled` is also used in all our other frontend apps, so I guess all of those will need to be changed to use `.govuk-frontend-supported` as well 👍  

https://github.com/search?q=org%3Aalphagov+js-enabled+repo%3Aalphagov%2Fgovuk_publishing_components+OR+repo%3Aalphagov%2Fstatic+OR+repo%3Aalphagov%2Fcollections+OR+repo%3Aalphagov%2Ffrontend+OR+repo%3Aalphagov%2Ffinder-frontend+OR+repo%3Aalphagov%2Femail-alert-frontend+OR+repo%3Aalphagov%2Fsmart-answers+OR+repo%3Aalphagov%2Fgovernment-frontend+OR+repo%3Aalphagov%2Fsmokey&type=code&p=1

Reply:
Thanks for adding this change in 👍